.github/workflows/pythonapp.yml

```diff
@@ -10,10 +10,10 @@ jobs:
   Test:
     runs-on: ${{ matrix.os }}
     strategy:
-      fail-fast: true
+      fail-fast: false
```

Comment:
Was this left over from testing?

Reply:
Nope!  I intentionally left this in there because it was helpful to see if only one test configuration (OS/python version/`psutil`) was failing or any of them.  The tests only took a few seconds to run anyways, so it didn't seem like a burden to keep the `fail-fast` option disabled.